src/cargo-fuzz.rs

```diff
@@ -0,0 +1,15 @@
+use std::env;
+use std::process;
+
+
+fn main() {
```

Comment:
> IMO cargo fuzz should also build the example file, like cargo run --example

For example, if I was in a directory of a crate, and typed `cargo fuzz`, it'd build the crate and run `afl-fuzz` with the built binary as a parameter?